### PR TITLE
OTJ: Ghired, Selvala, Sterling Supplier

### PR DESCRIPTION
**Pull request by dracontes**
Tested card scripts for my best reckoning of:
- [Ghired, Mirror of the Wilds](https://www.magicspoiler.com/mtg-spoiler/ghired-mirror-of-the-wilds/)
- [Selvala, Eager Trailblazer](https://www.magicspoiler.com/mtg-spoiler/selvala-eager-trailblazer/)
- [Sterling Supplier](https://www.magicspoiler.com/mtg-spoiler/sterling-supplier/)

Minor fixes:
- Updating AI deckbuilding tags on Selvala, ~At Knifepoint~, and Wanted Griffin as requested.

### CODE REVIEW

**Review by tool4ever — APPROVED** [review]
nice work

File: forge-gui/res/cardsfolder/upcoming/selvala_eager_trailblazer.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Selvala, Eager Trailblazer
+ManaCost:2 G W
+Types:Legendary Creature Elf Scout
+PT:4/5
+K:Vigilance
+T:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a creature spell, create a 1/1 red Mercenary creature token with "{T}: Target creature you control gets +1/+0 until end of turn. Activate only as a sorcery."
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ r_1_1_mercenary_tappump | TokenOwner$ You
+A:AB$ ChooseColor | Cost$ T | AILogic$ MostProminentInComputerDeck | SubAbility$ DBMana | SpellDescription$ Choose a color. Add one mana of that color for each different power among creatures you control.
+SVar:DBMana:DB$ Mana | Produced$ Chosen | Amount$ X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearChosenColor$ True
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/selvala_eager_trailblazer.txt:10]: I don't think you need/want this Cleanup line

File: forge-gui/res/cardsfolder/upcoming/selvala_eager_trailblazer.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+SVar:X:Count$DifferentPower_Creature.YouCtrl
+DeckHas:Ability$Token & Type$Mercenary
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/selvala_eager_trailblazer.txt:12]: & Color$Red

**Review by Northmoc — DISMISSED** [review]
👍 Nice!

File: forge-gui/res/cardsfolder/upcoming/at_knifepoint.txt [hunk 3]
```diff
@@ -4,6 +4,6 @@ Types:Enchantment
 S:Mode$ Continuous | Affected$ Creature.Outlaw+YouCtrl | AddKeyword$ First Strike | Condition$ PlayerTurn | Description$ As long as it's your turn, outlaws you control have first strike. (Assassins, Mercenaries, Pirates, Rogues, and Warlocks are outlaws.)
 T:Mode$ CommitCrime | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigToken | ActivationLimit$ 1 | TriggerDescription$ Whenever you commit a crime, create a 1/1 red Mercenary creature token with "{T}: Target creature you control gets +1/+0 until end of turn. Activate only as a sorcery." This ability triggers only once each turn.
 SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ r_1_1_mercenary_tappump | TokenOwner$ You
-DeckHas:Ability$Token & Type$Mercenary
+DeckHas:Ability$Token & Type$Mercenary & Color$Red
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/at_knifepoint.txt:7]: Don't need this one because card is already red!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**